app/sections/about/about.component.ts

```diff
@@ -18,53 +18,33 @@ import {transformResponse} from '../../shared/tools/response.tools';
   pipes: [MarkdownPipe]
 })
 export class About implements OnActivate {
-  private content: any;
-  private submenuItems: any[] = [];
+  private content: NodePageContent;
+  private submenuItems: NodePageContent[] = [];
 
   constructor(private _routerData: RouteData,
               private _params: RouteParams,
-              private _contentful: ContentfulService) {
+              private _contentfulContent: ContenfulContent) {
   }
 
   routerOnActivate(to: ComponentInstruction, from: ComponentInstruction): any {
```

Comment:
yes, I agree that we should stick with one convention, and the `OnInit` is the best choice here. 

About `Promise` we don't need it, I was trying to simulate something similar to angular1 resolve, but it's not necessary here